### PR TITLE
Update design tokens

### PR DESCRIPTION
**Pull request by tuentisre**
This PR was automatically created by the import-design-tokens GitHub Action.

### CODE REVIEW

**Review by dagonco — APPROVED** [review]

**Review by yamal-alm — APPROVED** [review]

File: library/src/main/res/values/themes_movistar_autogenerated.xml [hunk 1]
```diff
@@ -125,7 +125,9 @@
 		<item name="tabsLabelFont">?font_family_medium</item>
 		<item name="linkFont">?font_family_medium</item>
 		<item name="title1Font">?font_family_medium</item>
+		<item name="title2Font">?font_family_regular</item>
```
Comment (AnaMontes11) [inline library/src/main/res/values/themes_movistar_autogenerated.xml:128]: title2 in Movistar is bold

File: library/src/main/res/values/themes_vivo_autogenerated.xml [hunk 2]
```diff
@@ -138,7 +140,9 @@
 		<item name="tabsLabelTextStyle">normal</item>
 		<item name="linkTextStyle">normal</item>
 		<item name="title1TextStyle">normal</item>
+		<item name="title2TextStyle">normal</item>
```
Comment (AnaMontes11) [inline library/src/main/res/values/themes_vivo_autogenerated.xml:143]: light

File: library/src/main/res/values/themes_o2_autogenerated.xml [hunk 3]
```diff
@@ -138,7 +140,9 @@
 		<item name="tabsLabelTextStyle">normal</item>
 		<item name="linkTextStyle">normal</item>
 		<item name="title1TextStyle">normal</item>
+		<item name="title2TextStyle">normal</item>
```
Comment (AnaMontes11) [inline library/src/main/res/values/themes_o2_autogenerated.xml:143]: Light

File: library/src/main/res/values/themes_o2_autogenerated.xml [hunk 4]
```diff
@@ -138,7 +140,9 @@
...
 		<item name="indicatorTextStyle">normal</item>
+		<item name="navigationBarTextStyle">normal</item>
```
Comment (AnaMontes11) [inline library/src/main/res/values/themes_o2_autogenerated.xml:145]: Medium

File: library/src/main/res/values/themes_vivo_autogenerated.xml (hunk 2 above)
Comment (dagonco) [reply library/src/main/res/values/themes_vivo_autogenerated.xml:143]: Those parameters are correct. Android just check if the style of the text is bold, italic or **normal**.
Normal means, no bold, no italic.

What causes the text to use the light font is "title2Font" attribute.

Sounds weird I know, but thanks for noticing! 😇

**Review by AnaMontes11 — APPROVED** [review]
LGTM!

**Pull request merged**